src/pages/landing/ContentSection/QuotesCarousel.tsx

```diff
@@ -0,0 +1,256 @@
+
+  @media screen and (max-width: ${MOBILE_MAX_WIDTH}) {
+    min-width: 100%;
+  }
+`;
+
+const CarouselContainer = styled(Carousel)`
+  position: relative;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  width: 42rem;
+  margin: 6rem auto;
+
+  & .carousel.carousel-slider {
+    overflow: unset;
+  }
+
+  & .slider-wrapper.axis-horizontal {
+    overflow: unset;
+    width: 36rem;
+  }
+
+  & .carousel .slider-wrapper {
+    margin: unset;
+  }
+
+  & .slider {
+    align-items: center;
+  }
+
+  & .slide.selected {
+    opacity: 1;
+    transform: unset;
+    filter: unset;
+    z-index: 1;
+
+    & ${QuoteCard} {
+      backdrop-filter: blur(30px);
+      background: rgba(255, 255, 255, 0.05);
+    }
+  }
+
+  & .slide {
+    opacity: 0;
+    transform: scale(0.3) translate3d(0, 0, 0); // translate3d fix blur cropped on Safari
+    transition: opacity 0.3s, transform 0.3s;
+    filter: blur(7px);
+
+    &:has([data-previous='true']),
+    &:has([data-next='true']) {
+      opacity: 0.75;
+      transform: scale(0.75) translate3d(0, 0, 0); // translate3d fix blur cropped on Safari
+    }
+  }
+
+  & .control-arrow.control-prev {
+    height: 7.2rem;
+    width: 7.2rem;
+    top: unset;
+    bottom: -8.2rem;
+    left: 35%;
+    transform: translateX(-100%);
+    opacity: 1;
+    background-image: url('/img/arrow_left.svg') !important;
+
+    &:hover {
+      opacity: 0.8;
+    }
+  }
+
+  & .control-arrow.control-next {
+    height: 7.2rem;
+    width: 7.2rem;
+    top: unset;
+    bottom: -8.2rem;
+    left: 65%;
+    opacity: 1;
+    background-image: url('/img/arrow.svg') !important;
+
+    &:hover {
+      opacity: 0.8;
+    }
+  }
+
+  & .control-next::before,
+  .control-prev::before {
+    opacity: 0;
+  }
+
+  & .control-arrow.control-prev:hover,
+  .control-arrow.control-next:hover {
+    background-color: transparent;
+  }
+
+  @media screen and (max-width: ${MOBILE_MAX_WIDTH}) {
+    width: 100%;
+
+    & .control-arrow.control-prev {
+      bottom: -8.5rem;
+    }
+
+    & .control-arrow.control-next {
+      bottom: -8.5rem;
+    }
+
+    & .carousel .slider-wrapper {
+      width: 100%;
+    }
+
+    & .slider-wrapper.axis-horizontal {
+      padding: 0 1rem;
+    }
+  }
+`;
+
+const Quote = styled.span`
+  font-size: 1.5rem;
+  line-height: 2.5rem;
+  letter-spacing: 0.1rem;
+  font-weight: 500;
+  color: #fff;
+  text-align: left;
+
+  @media screen and (max-width: ${MOBILE_MAX_WIDTH}) {
+    padding-top: 3rem;
+  }
+`;
+
+const Divider = styled.div`
+  width: 100%;
+  height: 1px;
+  background-color: rgba(255, 255, 255, 0.2);
+  margin: 3rem 0 2rem;
+`;
+
+const AuthorContainer = styled.a`
+  width: 100%;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  gap: 2rem;
+  text-decoration: none;
+`;
+
+const Avatar = styled.img`
+  // override the default slide image
+  width: 4.5rem !important;
+  height: 4.5rem !important;
```

Comment:
is this necessary?
i would recommend adding a className to avoid using `!important`

Reply:
https://github.com/defi-wonderland/web/pull/113/commits/b4546936526107250dcf4e0b6a9606481cef7474

Reply:
geat! but you forgot to delete all `!important`s 😅

Reply:
https://github.com/defi-wonderland/web/pull/113/commits/3c7229e2e799ead87f8e39e0e93d1c551f384d01